### ISSUE TITLE
[Contribution Idea] Mini-Log

### ISSUE THREAD

**Issue by unmoving-javi** [Ajarmar/universal-pokemon-randomizer-zx#277]
**Is your idea a new setting, a change to an existing setting, or something else? Please describe clearly:** 
A log file alternative that would _only_ give information on the starter Pokemon, their abilities, and their types. Only things that would be found out minutes into the game. 

**What problem would your contribution solve? Please describe:**
3DS games take much longer to load + randomize, and if a starter is randomized to something like Cosmog, which doesn't learn anything but Splash, you'd have to potentially eject your SD card and re-randomize the whole thing, which would take much longer than generating a mini-log with easy-to-find data.

This would also allow for a small preview of what you're getting into without providing any significant advantage in a race (and might even be a nice thing to hand out to all participants beforehand).

**Comment by tom-overton** [comment]
I'm not quite sure what issue this solves. If you feel like you're going to regret your starter choice, you can simply save beforehand and reload if you don't like what you got (mind you, this *still* takes forever in Sun/Moon). You also have three different starters to choose from, so if one of them is a bit of a lemon, there's likely still at least *one* decent starter. And even if *all* of them are terrible, you can lose to your rival and still progress the game in all games except RSE and ORAS, so even in this extreme case there's a bit of leeway to catch something decent on the first route.

In short, I don't see a huge benefit to providing a preview of the starters like this. There might be a good use case for a trimmed down log, but I'm not sure it's this case. Also note that #246 was an idea about outputting logs as JSON; if this idea was implemented, you could theoretically write a tool that just parses the JSON and spits out whatever subset of the log you wanted, including the starter information. However, I don't actually know where the OP of that issue is in terms of implementing it, since we haven't heard anything from them since July.
